<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSetDlg</class>
 <widget class="QDialog" name="CSetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>音效设置</string>
    </attribute>
    <widget class="QRadioButton" name="diy1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>自定义</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="bgmusicswitch">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>背景音乐</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="soundswitch">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>音效</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>音效设置</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="default1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>71</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>默认</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="bgmusicpath">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string> 游戏背景音乐设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browser1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>170</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览...</string>
     </property>
    </widget>
    <widget class="QSlider" name="volumechange">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string> 游戏背景音乐音量</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>主题设置</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>宝石图片</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="default2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>默认</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="bgpicpath">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="acceptRichText">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="diy2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>自定义</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>背景图片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browser2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览...</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="style2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>71</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>风格2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="style1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>风格1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>姓名设置</string>
    </attribute>
    <widget class="QTextEdit" name="nameEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请输入您的游戏昵称：</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>CSetDlg</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CSetDlg</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browser1</sender>
   <signal>clicked()</signal>
   <receiver>CSetDlg</receiver>
   <slot>browser1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browser2</sender>
   <signal>clicked()</signal>
   <receiver>CSetDlg</receiver>
   <slot>browser2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cancel()</slot>
  <slot>save()</slot>
  <slot>browser1()</slot>
  <slot>browser2()</slot>
 </slots>
</ui>
